internal/namespaces/help/output.go

```diff
@@ -0,0 +1,69 @@
+package help
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+func helpOutput() *core.Command {
+	longText := `Output formatting in the CLI
+
+Human output
+
+- Standard human output
+
+	scw instance server list                       
+	ID                                    NAME               TYPE    STATE    ZONE      PUBLIC IP
+	088b01da-9ba7-40d2-bc55-eb3170f42185  scw-cool-franklin  DEV1-S  running  fr-par-1  51.15.251.251
+
+- Human with column selection
+
+You can select the columns that you want to print with commands that return a list 
+
+	scw instance server list -o human=Name,PublicIP
+	NAME                                            PUBLIC IP
+	scw-cool-franklin                               51.15.251.251
+
+JSON output
+
+- Standard JSON output
+
+	scw config dump -o json
+	{"access_key":"SCWXXXXXXXXXXXXXXXXX","secret_key":"11111111-1111-1111-1111-111111111111","default_organization_id":"11111111-1111-1111-1111-111111111111","default_region":"fr-par","default_zone":"fr-par-1","send_telemetry":true}
+
+- Pretty JSON output
+
+	scw config dump -o json=pretty
+	{
+	  "access_key": "SCWXXXXXXXXXXXXXXXXX",
+	  "secret_key": "11111111-1111-1111-1111-111111111111",
+	  "default_organization_id": "11111111-1111-1111-1111-111111111111",
+	  "default_region": "fr-par",
+	  "default_zone": "fr-par-1",
+	  "send_telemetry": true
+	}
+
+Template output
+
+You can use Go template to manipulate the output of a command and create a custom rendering of your resources. 
+Visit https://golang.org/pkg/text/template/ to learn more about Go template format.
+
+	scw instance server list -o template="{{ .Name }}---{{ .Status }}"
+	TODO, show example here
+
+`
+	return &core.Command{
+		Short:                "Get help about how the CLI output works",
+		Long:                 longText,
+		Namespace:            "help",
+		Resource:             "output",
+		AllowAnonymousClient: true,
+		ArgsType:             reflect.TypeOf(struct{}{}),
```

Comment:
Yes otherwise it panics:

```
$ go run cmd/scw/main.go help output  
panic: reflect: New(nil)

goroutine 1 [running]:
reflect.New(0x0, 0x0, 0xc000131950, 0x10051d0, 0x17ee9d8)
	/usr/local/Cellar/go/1.15/libexec/src/reflect/value.go:2372 +0xc5
github.com/scaleway/scaleway-cli/internal/core.run(0x18130e0, 0xc000406810, 0xc000482f00, 0xc00042e000, 0x1b86280, 0x0, 0x0, 0x0, 0x1b86280, 0x0, ...)
	/Users/sieben/workspace/scaleway-cli/internal/core/cobra_utils.go:105 +0x4d
github.com/scaleway/scaleway-cli/internal/core.cobraRun.func1(0xc000482f00, 0x1b86280, 0x0, 0x0, 0x0, 0x0)
	/Users/sieben/workspace/scaleway-cli/internal/core/cobra_utils.go:47 +0xa45
github.com/spf13/cobra.(*Command).execute(0xc000482f00, 0x1b86280, 0x0, 0x0, 0xc000482f00, 0x1b86280)
	/Users/sieben/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001a4f00, 0xc0004262f9, 0x16f2474, 0x5)
	/Users/sieben/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x30b
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/sieben/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/scaleway/scaleway-cli/internal/core.Bootstrap(0xc0000cded0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/sieben/workspace/scaleway-cli/internal/core/bootstrap.go:233 +0xc56
main.main()
	/Users/sieben/workspace/scaleway-cli/cmd/scw/main.go:47 +0x293
exit status 2
```